### PR TITLE
Add KeKsHost provider instructions

### PR DESCRIPTION
**Pull request by HostRK1**
* Update providers.json

Update informations about using Geyser on KeKsHost (hosting provider).

Correct copy/paste error

* Update providers.json

* Update _data/providers.json

### CODE REVIEW

File: _data/providers.json [hunk 1]
```diff
@@ -249,6 +249,11 @@
       "url": "https://humbleservers.com/",
       "description": "Set the Bedrock port to the Java server's port, or to one of the two extra ports, and connect with that port. If the subdomain doesn't work, use your regular numbered IP address."
     },
+    {
+      "name": "KeKsHost",
+      "url": "https://kekshost.com/en/",
+      "description": "Get Geyser as a plugin, as in the setup guide. Use the 19131 port and the server ip and connect with the same IP and port as you would on Java. You can also ask for a new port."
```
Comment (onebeastchris) [inline _data/providers.json:255]: Use port 19131, but then connect with the same port as on Java?
Sorry, that doesn't make sense. Additionally, please use the same layout as other entries.

**Review by onebeastchris — CHANGES_REQUESTED** [review]

File: _data/providers.json (hunk 1 above)
Comment (HostRK1) [reply _data/providers.json]: I fixed it

File: _data/providers.json [hunk 2]
```diff
@@ -249,6 +249,11 @@
       "url": "https://humbleservers.com/",
       "description": "Set the Bedrock port to the Java server's port, or to one of the two extra ports, and connect with that port. If the subdomain doesn't work, use your regular numbered IP address."
     },
+    {
+      "name": "KeKsHost",
+      "url": "https://kekshost.com/en/",
+      "description": "Get Geyser as a plugin, as in the setup guide. Use the 19131 port and the server ip and connect with the same IP and port 19132. You can also ask for a new port."
```
Comment (onebeastchris) [inline _data/providers.json:255]: Now you're asking people to set the port 19131, but connect with 19132? 
This still doesn't seem right. Additionally, please adjust the text flow to make it similar to the other texts (aka, remove the "get geyser as a plugin..." in the beginning)

**Review by onebeastchris — CHANGES_REQUESTED** [review]

File: _data/providers.json [hunk 3]
```diff
@@ -249,6 +249,11 @@
       "url": "https://humbleservers.com/",
       "description": "Set the Bedrock port to the Java server's port, or to one of the two extra ports, and connect with that port. If the subdomain doesn't work, use your regular numbered IP address."
     },
+    {
+      "name": "KeKsHost",
+      "url": "https://kekshost.com/en/",
+      "description": "Use the 19132 port and the server ip and connect with the same IP and port 19132. You can also ask for a new port."
```
Comment (onebeastchris) [inline _data/providers.json:255]: 19132 is the default port anyways, so users don't have to specify it..

```suggestion
      "description": "Use the default bedrock port (19132) and the server ip to connect on Bedrock edition. No changes to the Geyser config are needed. You can also ask for a new port and use it instead."
```

**Review by onebeastchris — CHANGES_REQUESTED** [review]

**Review by onebeastchris — APPROVED** [review]

**Pull request merged**